src/repositories/record_repository.rs

```diff
@@ -38,22 +39,25 @@ impl PostgresRepository {
         let mut temperatures = Vec::with_capacity(datas_len);
         let mut pressures = Vec::with_capacity(datas_len);
         let mut humidities = Vec::with_capacity(datas_len);
+        let mut dates = Vec::with_capacity(datas_len);
 
         datas.iter().for_each(|data| {
             temperatures.push(data.temperature);
             pressures.push(data.pressure);
             humidities.push(data.humidity);
+            dates.push(data.date);
         });
 
         sqlx::query(
             r#"
-            INSERT INTO records (temperature, pressure, humidity) 
-            SELECT * FROM UNNEST ($1,$2,$3);
+            INSERT INTO records (temperature, pressure, humidity, date) 
+            SELECT * FROM UNNEST ($1,$2,$3,$4);
         "#,
         )
         .bind(temperatures)
         .bind(pressures)
         .bind(humidities)
```

Comment:
there[ has to be a better way](https://docs.rs/sqlx/latest/sqlx/struct.QueryBuilder.html#method.push_values) 🤔